### PR TITLE
updated prod success verbiage.

### PR DESCRIPTION
**Pull request by hansknecht**

### CODE REVIEW

**Comment by GreerSamuel** [comment]
@hansknecht  I like your additions.

You removed the initial premise underneath the first question: Getting the right work done at the right time

Was that intentional?

File: ProdSuccess.md [hunk 1]
```diff
@@ -1,40 +1,44 @@
 # What is required to deliver a product successfully?
```
Comment (GreerSamuel) [inline ProdSuccess.md:2]: Initial premise was removed. 

Getting the right work done at the right time

Was that intentional?

File: ProdSuccess.md [hunk 2]
```diff
@@ -1,40 +1,44 @@
...
-- Defining the appropriate frame of reference for the objectives at hand
-- Evaluating the full scope of work necessary to accomplish, including:
+- Defining the frame of reference for the objectives at hand
+- Evaluating the full scope of work to accomplish, including:
 	- Education
 	- Supporting team members
 	- Including discovery, hand-off, and transition time
 	- Providing a sufficient accounting for and defense of the work being accomplished
 	- Sufficiently justifying the value the team brings to the table
-- Tackling turd tasks
```
Comment (GreerSamuel) [inline ProdSuccess.md:26]: Your commentary is more professional, though mine had alliteration.

File: ProdSuccess.md (hunk 1 above)
Comment (hansknecht) [reply ProdSuccess.md:2]: I added that back.

**Pull request merged**